.github/workflows/sync-release-branch-2023a.yml

```diff
@@ -2,10 +2,8 @@
 
 name: Sync downstream release branch with upstream's
 on:
- # Triggers the workflow every Tue at 8 A.M
- schedule:
-   - cron: "0 8 * * 2"
-
+  pull_request:
+    branches: [ main ]
```

Comment:
This one should be reverted to:
 schedule:
   - cron: "0 8 * * 2"

Reply:
yep, fixed